.github/workflows/CI.yaml

```diff
@@ -12,11 +12,14 @@ env:
 
 jobs:
   check:
-    runs-on: self-hosted
+    runs-on: [self-hosted, cachepot]
     steps:
       - name: "ACTIONS: Checkout"
         uses: actions/checkout@v2
 
+      - name: "Install: Set cargo path"
+        run: export PATH='/tmp/cargo/bin:$PATH'
```

Comment:
does it work? I found this note - https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#adding-a-system-path